content/post/frontend_talks_2017.md

```diff
@@ -0,0 +1,142 @@
+All main talks lasted 45 minutes, 30 for the talk and 15 for questions and answers while the lightning talks lasted for 15 minutes and there was no time for questions and answers.
+
+## Morning session
+
+### The multithreaded web: a tale of workers, _Ramón Guijarro_
+
+In the first morning talk, [Ramón Guijarro](https://twitter.com/soyguijarro), frontend developer at Tuenti, talked about how to develop multi-threaded applications in the web using workers.
+
+Ramón started his talk mentioning how the release of the iPhone and afterwards the App Store started a change in user expectations. So we've had to move from web pages to web apps to meet those user expectations on the web.
+
+Users expect their apps to have smooth animations and have a fast response, which can be achieved optimizing the JavaScript code to run at 60fps either splitting big time consuming tasks into chunks or moving that to another thread to stop blocking the UI.
+
+Users also expect their apps to work offline and sync in the background, as well as a very deep system integration: push notifications, geolocation, etc. All that can be achieved using service workers.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/qXzCcDePOtA" frameborder="0" allowfullscreen></iframe>
+
+### Web accessibility. How to make a frontend for everyone?, _Juanjo Montiel_
+
+In the second morning talk, [Juanjo Montiel](https://twitter.com/kastwey), software engineer at Pasiona, talked about web accessibility and showed us the difficulties visually impaired people face browsing the web.
+
+By just asking us to raise our hands to answer a question Juanjo showed us one of the problems that visually impaired people had that can be solved with accessible technology, such as a mobile app for voting instead of just raising our hands.
+Like in the previous example, visually impaired people have a lot of problems browsing the web and using a computer but Juanjo explains how to make our websites accessible to make it easier for them.
+
+Visually impaired people use softwares called screen readers that read everything in the computer and tell it to the person. But that works well only if the web page is accessible!
+
+With just a few guidelines we can make our web pages accessible too:
+
+* Provide a good structure, visually impaired people lack the visual position information.
+* Make good use of title elements (`h1`, `h2`, ...), list elements, link elements, ...
+* Label all the images that provide information to the reader with the `alt` attribute. Those images that are decorative should have the `alt` attribute but it must be empty.
+* Use labels for your form controls and fieldsets to group your radio buttons.
+* Don't use divs for everything if you can, use buttons, links, selects, etc. But if you can't, use the `role` aria attribute.
+* Always provide an alternative to drag and drop.
+
+And many more that you can find out watching the full talk.
```

Comment:
I'd provide context as an aside note just below the paragraph you explained that it is used screen readers. Example:
«Juanjo showed us how his reader works, and we was amazed about the reading speed he is used to. Don't miss that funny moment in the video (with time-link if you want)»